scratch/app-drive.jsonc

```diff
@@ -0,0 +1,72 @@
+[ // only using array here to have valid in the file. In reality, only the first object here would be returned. Then then the second one would be returned upon fetching from `source`
+
+    {
+        "type": "view",
+        "source": "views/file-list"
+    },
+
+    // Response
+    {
+        "type": "view",
+        "bindings": [
+            {
+                "type": "select",
+                "source": "lookup/available-drives"
+            },
+
+            // Response of lookup, or alternatively embedded right here instead of using a lookup
+            {
+                "type": "select",
+                "bindings": [
+                    {
+                        "label": "My Drive",
+                        "submit": {
+                            "path": "views/file-list/refresh",
```

Comment:
If the App responds with a view, and doesn't specify where it should be rendered, the frontend will implicitly decide that it should replace the view of its closest "fetched" view parent. In this case, it is the view originally defined at `/views/file-list`.